### PR TITLE
027 - homepage revamp, slider introduced

### PR DESCRIPTION
**Pull request by garySiladi**
This PR introduces the Slider element, also gives the homepage a basic layout according to the newest wireframes. Added tests to these new elements. The CSS and logic are not final, they need further improvement according to the tasks coming up.

### CODE REVIEW

File: src/components/app/app.js [hunk 1]
```diff
@@ -1,18 +1,14 @@
...
 // App needs to be a class in order to allow hot-reloading
 // that's why we disable react/prefer-stateless-function
 class App extends React.Component { // eslint-disable-line react/prefer-stateless-function
   render() {
     return (
       <div className="app">
-        <p>This is app page</p>
-        <Button bsStyle="success" bsSize="large">bootstrap is working</Button>
-        <Navigation />
+        <HomeContainer />
```
Comment (varholak-peter) [inline src/components/app/app.js]: We will not have this in App but it is fine for now.

File: src/components/home-container/home-container.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    margin: 0 5px 0 100px;
+  }
+
+  & > div {
+    margin: auto;
+    width: 100%;
+  }
+
+  &-slider {
+    font-family: 'Oxygen', sans-serif;
```
Comment (varholak-peter) [inline src/components/home-container/home-container.css]: lel

File: src/components/home-container/home-container.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        />
+        <Slider
+          data={shelves[0].items}
+          className="home-slider"
+          sliderTitle={shelves[0].title}
+        />
+        <Slider
+          data={allSeries ? allSeries.items : []}
+          className="home-slider"
+          {...sliderProps}
```
Comment (varholak-peter) [inline src/components/home-container/home-container.js]: This is a hacky way, we want a better approach in the future.

File: src/components/slider/slider.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    sliderTitle: 'Slider Title',
+  };
+  props: SliderStructure
+  render() {
+    return (
+      <div className={this.props.className}>
+        <div className="slider-title">{this.props.sliderTitle}</div>
+        <div className="slider-items">
+          {this.props.data.map(item =>
+            <Link className="slider-item" key={item.title} to={`/episode/${item.id}`}>
```
Comment (varholak-peter) [inline src/components/slider/slider.js]: We do want to decouple this onto separate component

**Comment by garySiladi** [comment]
Close in favor of #23

**Pull request closed without merging**